packages/create-rsbuild/template-react-ts/src/env.d.ts

```diff
@@ -1 +1,8 @@
 /// <reference types="@rsbuild/core/types" />
+
+declare module "*.svg?react" {
+  import React from "react";
+  const ReactComponent: React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
+  export default ReactComponent;
+  export { ReactComponent };
```

Comment:
I would prefer to remove the named export, since this is not the default behavior of `@rsbuild/plugin-svgr`. When users set `exportType: 'named'`, they can refer to [this guide](https://rsbuild.rs/plugins/list/plugin-svgr#type-declaration) to update the type declaration.

```suggestion
```